fluxc/src/main/java/org/wordpress/android/fluxc/store/PostStore.java

```diff
@@ -605,10 +605,19 @@ private void handleFetchedPostList(FetchPostListResponsePayload payload) {
             Map<Long, PostModel> posts = getPostsByRemotePostIds(postIds, site);
             for (PostListItem item : payload.postListItems) {
                 PostModel post = posts.get(item.remotePostId);
-                // Dispatch a fetch action for the posts that are changed, but not for posts with local changes
-                // as we'd otherwise overwrite and lose these local changes forever
-                if (post != null && !post.getLastModified().equals(item.lastModified) && !post.isLocallyChanged()) {
-                    mDispatcher.dispatch(PostActionBuilder.newFetchPostAction(new RemotePostPayload(post, site)));
+                if (post != null) {
```

Comment:
Can''t we check for the post being different at the beginning instead of doing it in both inner `if` statements? That'd simplify the inner `if` to just an `if (post.isLocallyChanged() { // do something } else { // do something else }`.

```suggestion
                if (post != null && !post.getLastModified().equals(item.lastModified)) {
```